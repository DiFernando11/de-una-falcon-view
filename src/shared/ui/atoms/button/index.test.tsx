import { render, screen } from '@testing-library/react';
import Button from '.';

describe('<Button />', () => {
  it('should renders correctly with text', () => {
    // Arrange
    const label = 'Hola mundo';
    // Act
    render(<Button>{label}</Button>);
    // Assert
    expect(screen.getByText(label)).toBeInTheDocument();
  });
});
