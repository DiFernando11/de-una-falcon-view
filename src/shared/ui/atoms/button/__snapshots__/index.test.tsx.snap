// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`<Button /> should match snapshot 1`] = `
<DocumentFragment>
  <button
    class="inline-flex items-center justify-center gap-2 rounded transition-all font-semibold bg-blue-600 text-white hover:bg-blue-700 text-base px-4 py-2"
  >
    Bot√≥n snapshot
  </button>
</DocumentFragment>
`;
